src/Middleware/HttpOverrides/src/ForwardedHeadersMiddleware.cs

```diff
@@ -213,11 +213,7 @@ public void ApplyForwarders(HttpContext context)
             // Host and Scheme initial values are never inspected, no need to set them here.
         };
 
-        var checkKnownIps = _options.KnownIPNetworks.Count > 0
-#pragma warning disable ASPDEPR005 // KnownNetworks is obsolete
-            || _options.KnownNetworks.Count > 0
-#pragma warning restore ASPDEPR005 // KnownNetworks is obsolete
-            || _options.KnownProxies.Count > 0;
+        var checkKnownIps = _options.KnownIPNetworks.Count > 0 || _options.KnownProxies.Count > 0;
```

Comment:
I might be missing something here but can't this just check on KnownProxies as that represents both?

Reply:
This is what it's doing, the names are `KnownNetworks` and `KnownIPNetworks`, `KnownProxies` is something else. Now it's only checking for `KnownIPNetworks` which is the newly introduced property. The previous extra check on `KnownNetworks` is removed.